app/assets/javascripts/ama_layout/desktop/header_menu.coffee

```diff
@@ -1,3 +1,8 @@
-$(document).ready ->
-  $('.menu-link').click ->
-    $('#menu').toggleClass('active')
\ No newline at end of file
+$(window).load ->
+  ww = $(window).width()
```

Comment:
We try to avoid abbreviations for variable names. `width` and `height` would be easier to read than `ww` and `wh`.